src/__experimental__/components/form/form.component.js

```diff
@@ -207,9 +207,16 @@ class FormWithoutValidations extends React.Component {
       this.triggerSubmit(ev, valid);
     } else {
       this.setState({ submitted: false });
+      this.regainFocusOnSubmit();
     }
   }
 
+  regainFocusOnSubmit() {
```

Comment:
Why is this needed, it would be better if we prevent `focus` from changing in the first place. Screen readers will read the element that is briefly focused otherwise.